### DISCUSSION TITLE
Nulleinspeisung (DLP)  mit zwei Wechselrichtern 50/50

### DISCUSSION THREAD

**Discussion by RicoBach68** [hoylabs/OpenDTU-OnBattery#1375] [General]
Ich betreibe 2 HM-1500 (mit vorgeschalteter Zendure HUB2000/AB2000) und mein größter Wunsch wär es, beide Wechselrichter einfach zu 50/50% mit einer DTUoB zu steuern. Die Daten kommen von einem Shelly 3EM Pro. 1000W Hausbedarf sollen also einfach zu je 500W auf beide Wechselrichter gleichmäßig verteilt werden. ZZ steuere ich nur eine Anlage und die zweite manuell, so gut es geht.
Steht so etwas auf deiner todo Liste oder wird es ein Wunsch bleiben? 
Danke für dieses tolle Projekt und für die vielen Stunden Arbeit!

**Comment by schlimmchen** [comment]
Siehe #1216.

Dein Wunsch, die Leistung speziell zu gleichen Teilen zu verteilen, ist nicht berücksichtigt. Gibt es dazu einen wichtigen Grund? Meiner Einschätzung nach sind andere Dinge, vor allem die möglichst schnelle Regelung, viel wichtiger, als die Leistung speziell aufzuteilen. Hauptsache ist meiner Meinung nach, dass mehrere WR so gesteuert werden, dass der Bedarf gedeckt wird, auch wenn ein WR ihn nicht alleine decken könnte.

**Reply by RicoBach68** [reply]
Das stimmt natürlich. Mein Beweggrund dafür ist, dass ich zwei identische Anlagen mit identischen Zendure AB2000 Akkus habe, die möglichst gleichmäßig entladen werden sollen um möglichst zu jedem Zeitpunkt 2x 1200W Ausgangsleistung abrufen zu können. Wird ein Akku wesentlich mehr belastet und läuft leer bleiben am Ende nur noch 1200W von einem Akku übrig und die andere Anlage schaltet ab.

**Reply by RicoBach68** [reply]
Hallo Helge, vielen vielen Dank für die neue Firmware 2024.11.20. Nach welchen Kriterien bekommen zwei Wechselrichter die Leistungsanforderung durch den DLP zugeteilt? Mit Priorität des ersten und Deckelung durch den zweiten? Oder wird die Leistungsanforderung sofort aufgeteilt? Eine relativ gleichmäßige Aufteilung wäre halt immer noch mein absoluter Wunsch für meine Konstellation.

**Reply by spcqike** [reply]
@RicoBach68 

siehe
https://github.com/hoylabs/OpenDTU-OnBattery/issues/1409

wenn ich es richtig verstehe, wird es aktuell so gesteuert, dass möglichst wenig Schritte / Änderungen / Übertragungen stattfinden, damit die Regelung schnellstmöglich reagiert.

Es ist geplant, künftig unterschiedliche Strategien einzubauen, so dass der Benutzer entscheiden kann, welche er will. 

Eine gleichmäßige Verteilung der Last auf allen Wechselrichtern führt halt zu einer langsameren Regelung. Ändert sich deine Last um 20W musst du 2 Wechselrichter um je 10W reduzieren, was nacheinander passiert.

**Reply by schlimmchen** [reply]
Leute... Ich geb mir durchaus Mühe mit der [Doku](https://opendtu-onbattery.net/firmware/dpl/#strategy). Bitte lest sie. Deine Frage ist dort hinreichend beantwortet.

Google kann das auch super übersetzen, falls das nötig ist.
